src/proteomes/components/landing-page/styles/landing-page.module.scss

```
@@ -0,0 +1,68 @@
+@import 'franklin-sites/src/styles/settings';
+@import 'franklin-sites/src/styles/colours';
+
+.landing-page {
+  margin-block: 2 * $global-margin;
+
+  p:last-of-type {
+    margin-bottom: 0;
+  }
+}
+
+.image-container {
+  display: grid;
+  place-items: center;
+}
+
+.statistics {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
+
+.chart {
+  text-align: center;
+}
+
+.entries-count {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+
+  p.stats-count {
+    display: flex;
+    gap: $global-margin;
+  }
+}
+
+.ref-prot-icon {
+  color: $colour-reference-proteome;
```

Try to use custom properties instead of SASS variables from now on. At the moment we have colours available, you can see them here https://github.com/ebi-uniprot/franklin-sites/blob/main/src/styles/_colours.scss#L58-L110

```suggestion
  color: var(--fr--color-reference-proteome);
```